### ISSUE TITLE
NoMethodError on parsing repository.

### ISSUE THREAD

**Issue by hirakiuc** [hirakiuc/tinybucket#46]
```
E, [2015-06-03T11:33:20.089064 #65520] ERROR -- : undefined method `_type=' for #<Tinybucket::Model::Repository:0x007fd7b1830460 @scm="git"> (NoMethodError)
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/base.rb:19:in `block in attributes='
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/base.rb:19:in `each'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/base.rb:19:in `attributes='
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/base.rb:13:in `initialize'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/parser/repo_parser.rb:5:in `new'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/parser/repo_parser.rb:5:in `convert'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/parser/base_parser.rb:5:in `process_response'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/faraday_middleware-0.9.1/lib/faraday_middleware/response_middleware.rb:32:in `block in call'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/faraday-0.9.0/lib/faraday/response.rb:57:in `on_complete'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/faraday_middleware-0.9.1/lib/faraday_middleware/response_middleware.rb:30:in `call'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/faraday-0.9.0/lib/faraday/rack_builder.rb:139:in `build_response'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/faraday-0.9.0/lib/faraday/connection.rb:377:in `run_request'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/faraday-0.9.0/lib/faraday/connection.rb:140:in `get'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/request.rb:31:in `request'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/request.rb:4:in `get_path'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/api/repo_api.rb:9:in `find'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/repository.rb:92:in `load_model'
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/concerns/reloadable.rb:13:in `load'
app.rb:21:in `block in <main>'
```

**Comment by hirakiuc** [comment]
[Repository model does not have `_type` attributes](https://github.com/hirakiuc/tinybucket/blob/master/lib/tinybucket/model/repository.rb#L7), but [Base model set all attributes in hash](https://github.com/hirakiuc/tinybucket/blob/master/lib/tinybucket/model/base.rb#L19).

This issue should be fixed.

**Comment by hirakiuc** [comment]
It seems like the attribute which added changed.

```
E, [2015-06-05T10:20:19.611965 #55044] ERROR -- : undefined method `type=' for #<Tinybucket::Model::Repository:0x007f974a0429f8> (NoMethodError)
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/base.rb:19:in `block in attributes='
(... snipped ...)
```

And same issue occurred on PullRequest Model.

```
/Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/base.rb:19:in `block in attributes=': undefined method `type=' for #<Tinybucket::Model::PullRequest:0x007fdba4692a30> (NoMethodError)
	from /Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/base.rb:19:in `each'
	from /Users/hirakiuc/Desktop/tool/bitbucket/.bundle/ruby/2.1.0/gems/tinybucket-0.1.3/lib/tinybucket/model/base.rb:19:in `attributes='
(... snipped ...)
```

**Comment by hirakiuc** [comment]
fixed by pull request #47
